.github/workflows/build.yaml

```diff
@@ -39,49 +48,48 @@ jobs:
           sudo apt-get install -y libayatana-appindicator3-dev
 
       - name: Build binary
-        run: GOOS=linux GOARCH=${{ env.DEB_ARCH }} go build -o ${{ env.APP_NAME }}
-          
-      - name: Create directory structure for .deb
-        run: |
-          mkdir -p ${{ env.APP_NAME }}_${{ env.APP_VERSION }}/{DEBIAN,${{ env.INSTALL_PATH }}}
-          mv ${{ env.APP_NAME }} ${{ env.APP_NAME }}_${{ env.APP_VERSION }}/${{ env.INSTALL_PATH }}/
-          cp -r assets ${{ env.APP_NAME }}_${{ env.APP_VERSION }}/${{ env.INSTALL_PATH }}/
-          echo "Package: ${{ env.APP_NAME }}
-          Version: ${{ env.APP_VERSION }}
-          Section: base
-          Priority: optional
-          Architecture: ${{ env.DEB_ARCH }}
-          Depends: libc6 (>= 2.14)
-          Maintainer: ${{ env.MAINTAINER_NAME }} ${{ env.MAINTAINER_EMAIL }}
-          Description: Wazuh Agent Status
-            A system tray application to monitor Wazuh Agent status." > ${{ env.APP_NAME }}_${{ env.APP_VERSION }}/DEBIAN/control    
-          
-      - name: Build .deb package
-        run: dpkg-deb --build ${{ env.APP_NAME }}_${{ env.APP_VERSION }}
+        run: GOOS=linux GOARCH=${{ env.DEB_ARCH }} go build -o ${{ env.APP_NAME }}-${{ matrix.target }}
 
       - name: Upload Linux binary
         uses: actions/upload-artifact@v4
         with:
-          name: ${{ env.APP_NAME }}-deb
-          path: ${{ env.APP_NAME }}_${{ env.APP_VERSION }}.deb
+          name: ${{ env.APP_NAME }}-${{ matrix.target }}
+          path: ./
+
+  release:
+    name: Release
+    permissions: write-all
+    needs:
+      - build-ubuntu
+      # - build-macos
+      # - build-windows
```

Comment:
Remove the comments if not used

Reply:
They will be used later, after the linux version will be stabilised

Reply:
Meaning that you could not test locally using arc?

Reply:
Don't have it yet locally